armTemplates/azuredeploy-blobforwarder.json

```diff
@@ -0,0 +1,154 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newRelicLicenseKey": {
+      "type": "string",
+      "metadata": {
+        "description": "New Relic License Key"
+      }
+    },
+    "newRelicEndpoint": {
+      "type": "string",
+      "defaultValue": "https://log-api.newrelic.com/log/v1",
+      "metadata": {
+        "description": "The Logs API endpoint used to send your logs to. By default, it is https://log-api.newrelic.com/log/v1 if your account is in the United States (US) region. Otherwise, if you're in the European Union (EU) region, you should use https://log-api.eu.newrelic.com/log/v1"
+      }
+    },
+    "storageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Azure Storage account."
+      },
+      "minLength": 3,
+      "maxLength": 24
+    },
+    "storageAccountSku": {
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS",
+        "Standard_RAGRS",
+        "Standard_ZRS",
+        "Premium_LRS",
+        "Premium_ZRS",
+        "Standard_GZRS",
+        "Standard_RAGZRS"
+      ],
+      "metadata": {
+        "description": "Sku on which to run the Azure Storage account."
```

Comment:
What is Sku? 🤔 it is an acronym? Do we know if Customers are going to understand the reference??


[Here](https://blog.hubspot.com/marketing/inclusive-language) some resources around inclusive language

Reply:
Good point, I will use the same parameter azure are using when we create a new service account `Redundancy`

Reply:
https://docs.microsoft.com/en-us/azure/templates/microsoft.storage/storageaccounts?tabs=bicep#sku